test/functional/rpc_psbt.py

```diff
@@ -174,8 +174,11 @@ def run_test(self):
             elif out['scriptPubKey']['addresses'][0] == p2pkh:
                 p2pkh_pos = out['n']
 
+        inputs = [{"txid": txid, "vout": p2wpkh_pos}, {"txid": txid, "vout": p2sh_p2wpkh_pos}, {"txid": txid, "vout": p2pkh_pos}]
+        addr = {self.nodes[1].getnewaddress(): 29.99}
```

Comment:
Nit: since this is an address and amount, how about "receiver_output"?

Reply:
Good idea. Changed it to `outputs`:
```diff
         inputs = [{"txid": txid, "vout": p2wpkh_pos}, {"txid": txid, "vout": p2sh_p2wpkh_pos}, {"txid": txid, "vout": p2pkh_pos}]
-        addr = {self.nodes[1].getnewaddress(): 29.99}
+        outputs = [{self.nodes[1].getnewaddress(): 29.99}]
```